README.md

```diff
@@ -739,6 +740,40 @@ More details are in [its own README](./cmd/scanblocks/README.md). The repository
 also includes a shell script, jsonarray2csv.sh, to convert the result into a
 comma-separated value (CSV) file.
 
+## Front End Development
+
+Make sure you have a recent version of [node and npm](https://nodejs.org/en/download/) installed.
+You may want to use the [node version manager (nvm)](https://github.com/creationix/nvm) for managing
+your node download and installation.
+
+From the dcrdata root directory, run the following command to install the node modules.
+
+`npm install`
```

Comment:
What is the output of this command?  As a node noob, I was searching around to see what it spat out.

And after this command, is it ok to run dcrdata?  Or does this just get the depends to build the bundle?

Reply:
The primary output is a directory called `node_modules`. I think that's all.